src/bikeracks/templates/scripts.js

```diff
@@ -502,12 +517,16 @@ BikeMap.prototype.vote = function(e) {
                     });
                 });
             }
-            // if the new vote is the same as the old vote, nothing happens
+            // if the new vote is the same as the old vote, UNVOTE
+            else {
```

Comment:
What happens on their first time voting?

Reply:
That's on line 530